frontend/utils/time.js

```diff
@@ -87,3 +91,16 @@ export function getTime (date: Date): string {
   const t = new Date(date)
   return `${t.getHours()}:${t.getMinutes()}`
 }
+
+// Returns the ratio of (the current day of the month) / (days in current month);
+// The current day / month are calculated with respect to the atDate parameter.
+export function cycleAtDate (atDate: string | Date, startDate: string | Date): any | number {
```

Comment:
> Remove any cop-out return value :P

:P Fine. Lol.

> Recommendation: decomplicificate the types - I think it's safe to make the parameter types be string, and use new Date().toISOString() where you pass in new Date() elsewhere

I did not do this because it was DRYer to do this internally. Can you double verify this?

> Also, what is the point of startDate? The comment says this function does the following:

Good catch. I totally didn't update this comment/annotation. Fixing.

The old way of doing this function only returned the fraction of the way through the month. Now it adds 1 to the final cycle integer returned for each month that passes, and 12 for every year that passes. This is necessary for `monthlyCycleTimer.js` to track the month changes as well as update the distributionEvents accordingly.